src/main/java/frc/robot/subsystems/conveyor/commands/bits/PostFlapTest.java

```diff
@@ -0,0 +1,39 @@
+package frc.robot.subsystems.conveyor.commands.bits;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.conveyor.Conveyor;
+
+public class PostFlapTest extends CommandBase {
+    private final Conveyor conveyor;
+    private final double expected;
+    private boolean isConnected = true;
+    private boolean wasConnected = true;
+
+    private int actual;
+
+    public PostFlapTest(Conveyor conveyor, double expected) {
+        this.conveyor = conveyor;
+        this.expected = expected;
+
+        addRequirements(conveyor);
+    }
+
+    @Override
+    public void execute() {
+        conveyor.setPower(0.5);
+        isConnected = conveyor.isPostFlapBeamConnected();
+
+        if (wasConnected && !isConnected){
```

Comment:
I think it's better to check the ball after it passes through the post flap beam, because that's how the logic works in the Conveyor subsystem class.